extensions/pomodoro/src/pomodoro-control-timer.tsx

```diff
@@ -107,8 +107,29 @@ const ActionsList = () => {
   );
 };
 
+const handleQuote = (): string => {
+  let markdownContent = "";
+  const quoteUrl = "https://api.quotable.io/quotes/random?limit=1";
+  // Get Quote here
+  const backupQuote = { content: "You did it!", author: "Unknown" }; // Backup quote if the API returns no data
+  const { isLoading, data } = useFetch(quoteUrl, { keepPreviousData: true }) as {
+    isLoading: boolean;
+    data: QuoteResponse;
+  };
+  if (!isLoading) {
+    markdownContent += data.length ? formatQuote(data[0]) : formatQuote(backupQuote);
+  }
+
+  return markdownContent;
+
+  function formatQuote(quote: { content: string; author: string }) {
+    // Format the quote in markdown:
+    return `> ${quote.content} \n>\n> &dash; ${quote.author}`;
+  }
+};
```

Comment:
Thanks for taking a look and the helpful feedback! This is much sleeker than how it was, so I've just updated it with your changes. I did have two comments on it:

1. I had to change the line below in order for VS Code to stop shouting at me (it was wanting `data.data` after the change) - I'm guessing this is what was intended, and looks to be working with this:

```diff
-- const { isLoading, data } = useFetch<{data: Quote[]}>(
+ const { isLoading, data } = useFetch<{Quote[]}>(
```

2. There is a small change of behaviour based on simplifying this method as you've suggested: we'll show the `"You did it!"` quote while the api request is loading, then it will change to the quote once ready. 
    * Previously, there will be no quote until the API request has resolved, then it will use the backup if no quote is returned. Now, depending on the API response speed, we may see a 'flicker' of the quote changing, but the API response is fast so I'm happy with this if you are! (I think this is personal preference anyway, some may prefer it not to be blank).
    * See gifs below demo'ing this:

### Before Change:

![Screen Recording 2024-02-20 at 16 46 53](https://github.com/raycast/extensions/assets/64075030/65b28f4c-ba18-4f84-83ba-46e2c744b946)


### With this new Change:

![Screen Recording 2024-02-20 at 16 42 41](https://github.com/raycast/extensions/assets/64075030/acc102cd-f906-458f-9e73-1c8fa2a718e8)